### PR TITLE
Make the index.html files the visible overview menu entry

### PR DESCRIPTION
**Pull request by SvenDowideit**
And move menu entry definition into a page that the user has no reason to navigate to

I've also made the 'Recipies' menu entry into another submenu, with the other recipes made visible in it.

Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

**Comment by SvenDowideit** [comment]
see 
- http://svendowideit.s3-website-us-east-1.amazonaws.com/registry/recipes/nginx/
- http://svendowideit.s3-website-us-east-1.amazonaws.com/registry/recipes/
- http://svendowideit.s3-website-us-east-1.amazonaws.com/registry/

File: docs/recipes/menu.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!--[metadata]>
++++
+title = "Docker Registry Recipes"
```
Comment (RichardScothern) [inline docs/recipes/menu.md]: could this just be 'recipes' ?  It's a bit crowded in the menu.

**Comment by RichardScothern** [comment]
One nit, but LGTM

File: docs/recipes/menu.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/recipes/menu.md]: yup, will do.

**Comment by SvenDowideit** [comment]
add a redirect from menu.md to index.md, and a redirect from the original page (like overview) to the new index.md

**Comment by SvenDowideit** [comment]
darn, can't use hugo aliases to generate a redirect page - when we move to nginx, this will be trivial, but for now..

**Comment by SvenDowideit** [comment]
aha, https://github.com/docker/docs-base/pull/253

**Comment by SvenDowideit** [comment]
@RichardScothern @thaJeztah 

This is the menu definition style we'll be moving towards - as it allows users to navigate to all the sub-URL's of topics.

**Comment by SvenDowideit** [comment]
oh, found 2 more.

File: docs/help.md [hunk 2]
```diff
@@ -21,4 +21,4 @@ If you want to report a bug:
 - be sure to first read about [how to contribute](https://github.com/docker/distribution/blob/master/CONTRIBUTING.md)
 - you can then do so on the [GitHub project bugtracker](https://github.com/docker/distribution/issues)
 
-You can also find out more about the Docker's project [Getting Help resources](https://docs.docker.com/opensource/get-help/).
+You can also find out more about the Docker's project [Getting Help resources](/opensource/get-help.md).
```
Comment (thaJeztah) [inline docs/help.md]: Downside here is that these will be broken on GitHub. Perhaps we should think more about how to do cross-repo linking. I know we discussed making these links to the GitHub repository and rewrite them to relative links during the docs build.

Comment (SvenDowideit) [inline docs/help.md]: we definitely need to work on this - the norm atm, is to link to the .md file, and work towards making the rendered version useful enough that everyone prefers to read there.

**Comment by thaJeztah** [comment]
LGTM

**Pull request merged**